Modules/vc-module-pricing/VirtoCommerce.PricingModule.Web/Scripts/blades/assignment-list.js

```diff
@@ -159,7 +161,12 @@ angular.module('virtoCommerce.pricingModule')
 
                         var selectedRows = $scope.gridApi.selection.getSelectedRows();
                         exportDataRequest.dataQuery.objectIds = [];
+                        exportDataRequest.dataQuery.isAllSelected = true;
+                        exportDataRequest.dataQuery.isAnyFilterApplied = false;
+
```

Comment:
I think we need to fill `keyword` (if it is specified) and fill `isAnyFilterApplied` based on its emptiness.